api/utils/debug.js

```diff
@@ -1,11 +1,17 @@
+const { DEBUG_CONSOLE = false } = process.env;
+
+const useDebugConsole =
+  (typeof DEBUG_CONSOLE === 'string' && DEBUG_CONSOLE?.toLowerCase() === 'true') ||
+  DEBUG_CONSOLE === true;
+
 const levels = {
   NONE: 0,
   LOW: 1,
   MEDIUM: 2,
   HIGH: 3,
 };
 
-let level = levels.HIGH;
+let level = useDebugConsole ? levels.HIGH : levels.NONE;
```

Comment:
instead of doing this? is it not better to just use the global logger?
```js
const { logger } = require('~/config');
```
https://github.com/danny-avila/LibreChat/tree/main/api/config/winston.js